libs/common/ui/src/theme/index.ts

```diff
@@ -41,6 +41,8 @@ declare module '@mui/material/styles' {
     rarity3: Palette['primary']
     rarity4: Palette['primary']
     rarity5: Palette['primary']
+
+    charSelected: Palette['primary']
```

Comment:
I am not sure if we need to have a separate color for this. I'd say we should use the default action color palette (success/warning/failure/info). 
In terms of color, I would actually recommend you to use "success"(greenish) instead of this yellow color, see how that looks.

Reply:
![image](https://github.com/user-attachments/assets/0de5f957-1d30-4d7d-a5ca-928998d59d6c)
I prefer the gold outlines, personally. The green isn't as visually distinct against the modal's background color and blends in with some of the namecards as well.

Reply:
I see. I am fine keeping it yellow, or warning for now